client/src/containers/TeamPage/logic/saga.ts

```diff
@@ -66,6 +70,38 @@ export function* updateField(props: Props) {
 	}
 }
 
+export function* searchPeople(props: Props) {
+	try {
+		yield put(actions.searchPeopleLoader());
+		const users = yield call(findUsersColleagues, props.id, props.match);
+		const searchResult: { data: WebApi.Entities.UserProfile; key: string; title?: string; }[] = [];
+		users.filter((el: WebApi.Entities.UserProfile) => {
```

Comment:
You shouldn't do search in saga. It is better to filter users in service
Also it is better to use forEach 
```suggestion
		users.forEach((el: WebApi.Entities.UserProfile) => {
```

Reply:
let it been as you wish